robottelo/ui/ldapauthsource.py

```diff
@@ -52,25 +60,14 @@ def create(self, name=None, server=None, ldaps=False, port=None,
 
     def search(self, name):
```

Comment:
Ok, I would ask for make this clear in form of comments/docstring. But this PR is already merged.

If you want to add this later so we can remember why it is needed.